rero_ils/modules/loans/jsonschemas/loans/loan-ils-v0.0.1.json

```diff
@@ -25,6 +25,30 @@
         }
       }
     },
+    "patron": {
+      "title": "Patron",
+      "type": "object",
+      "properties": {
+        "$ref": {
+          "title": "Patron URI",
+          "type": "string",
+          "pattern": "^https://ils.rero.ch/api/patrons/.+?$",
+          "default": "https://ils.rero.ch/api/patrons/"
+        }
+      }
+    },
+    "document": {
+      "title": "Document",
+      "type": "object",
+      "properties": {
+        "$ref": {
+          "title": "Document URI",
+          "type": "string",
+          "pattern": "^https://ils.rero.ch/api/documents/.+?$",
+          "default": "https://ils.rero.ch/api/documents/"
+        }
+      }
+    },
```

Comment:
I'm curious to know for what are these data used? invenio-circulation add them?

Reply:
Yes, those are the new fields added by invenio-circulation